embedded-bins/Makefile.variables

```diff
@@ -1,6 +1,6 @@
 alpine_version = 3.17
 alpine_patch_version = $(alpine_version).4
-golang_buildimage=docker.io/library/golang:$(go_version)-alpine$(alpine_version)
+golang_buildimage=docker.io/library/golang:$(go_version)-alpine3.18
```

Comment:
The problem only affects go compiler. We link statically with CGO_ENABLED=1.

We use alpine 3.17 other places and I thought it would be good to be conservative and continue with that for the stable branch.

Reply:
Another option to fix this is to explicitly request netgo, like this:

```patch
diff --git a/Makefile b/Makefile
index abc282d32..c7c7d9457 100644
--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@ EMBEDDED_BINS_BUILDMODE ?= docker
 TARGET_OS ?= linux
 BUILD_UID ?= $(shell id -u)
 BUILD_GID ?= $(shell id -g)
-BUILD_GO_FLAGS := -tags osusergo
+BUILD_GO_FLAGS := -tags netgo,osusergo
 BUILD_GO_LDFLAGS_EXTRA :=
 DEBUG ?= false
 
```